client/src/api/projects.js

```diff
@@ -0,0 +1,30 @@
+import httpClient from './httpClient';
+
+export const addProject = async (id, projectData) => {
+    const url = `/api/projects/add/${id}`;
+    const token = localStorage.getItem('jwtToken')
+    const authOptions = {
+        headers: {
+            Authorization: `Bearer ${token}`,
+        }
+    }
+    try {
+        const response = await httpClient.post(url, projectData, authOptions);
+        if (response.data.err) return { ...response.data }
+
+        if (typeof window !== 'undefined') {
```

Comment:
No problem with this -- it's just new to me. Is there a scenario in which the window would be undefined? I think it's just the DOM here, no?

Reply:
I think it's just some habit I picked up while learning this stuff. I think it's for if your script is running in a web browser or not. Could remove it

Reply:
Interesting. Yeah, I'm not sure. Is there a time when it wouldn't be running in a browser?

Reply:
I think the only times would be if it's running in node or with web workers. I could remove it for this project; I don't think it would be a big deal

Reply:
As @DevDelvo probably not a big deal, as this won't be running in node or non-browsers